src/DETHRACE/common/utility.c

```diff
@@ -195,8 +195,14 @@ int IRandomBetween(int pA, int pB) {
     int num;
     char s[32];
 
-    num = (pB + 1 - pA) * (rand() / (double)RAND_MAX) + pA;
-    return num;
+    num = rand();
+#if RAND_MAX == 0x7fff
+//  If RAND_MAX == 0x7fff, then `num` can be seen as a fixed point number with 15 fractional and 17 integral bits
```

Comment:
is it correct to think that this is the windows code path, and the #else is the *nix code path?

Reply:
I think you can. MinGW also does `0x7fff`.
I would keep the `#if RAND_MAX == 0x7fff` check though.